api/python/provisioner/__main__.py

```diff
@@ -245,9 +248,12 @@ def _main():
         auth_args_view['password'] = config.SECRET_MASK
 
     logger.debug(
-        f'Parsed arguments: auth={auth_args_view}, log={log_args_view}, '
-        f'cmd={parsed_args.cmd}, args={parsed_args.args}, '
-        f'kwargs={parsed_args.kwargs}'
+        "\nParsed arguments: \n"
```

Comment:
@andkononykhin2 I believe we have to go for best of both.
First line that is good for both humans as well as the log scrapers.
Any subsequent lines for users. Thus, we would like to have new-line.

Reply:
@divya-vijay Please create a follow-up task and put there that discussion since it includes good design start points. Generally I believe we need separate log destinations (files) for humans and tools. console should be for humans, rsyslog -  for tools (we should follow common approach here and don't mess it with our custom log solutions)